src/api/app/views/layouts/webui2/_breadcrumbs.html.haml

```diff
@@ -0,0 +1,10 @@
+%nav{"aria-label" => "breadcrumb"}
+  %ol.breadcrumb.bg-light
+    - @breadcrumbs.each do |breadcrumb|
+      %li.breadcrumb-item{class: "#{@breadcrumbs.last == breadcrumb ? 'active' : ''}", "aria-current" => "page"}
```

Comment:
what about using the way it is done in [HAML reference](http://haml.info/docs/yardoc/file.REFERENCE.html):

`%li.breadcrumb-item{class: @breadcrumbs.last == breadcrumb && 'active' , "aria-current" => "page"}`

:bowtie: